fullscreen.bs

```diff
@@ -295,18 +295,28 @@ attribute's getter must run these steps:
  <li><p>Return null.
 </ol>
 
+<p>A <a>document</a> is said to be a <dfn>simple fullscreen document</dfn> if its <a>top layer</a>
```

Comment:
What do we think about documents with a single fullscreen element in the top layer but a dialog above or beneath it?